GENERATORS/SIMULATIONS/FEEDBACK/example2.md

```diff
@@ -0,0 +1 @@
+THIS APP DEMONSTRATES HOW FEEDBACK NODE, LOOPS NODE & GOTO NODE WORKS.
```

Comment:
wtf is this? why is it all caps? 
delete this and write an actual explanation of what the example app is doing. i never want to see anything like this again please.